src/pages/Browse/index.spec.tsx

```diff
@@ -502,6 +502,6 @@ describe(__filename, () => {
 
     const root = render({ store, versionId: String(version.id) });
 
-    expect(root.find('title')).toHaveText(`Browse ${name}@${versionString}`);
+    expect(root.find('title')).toHaveText(`Browse ${name} @ ${versionString}`);
```

Comment:
You could maybe use `:` since this is the separator used in the compare title page

Reply:
Heh, that's what QA suggested, too. They are conveying different things, though. The browse title is conveying *browse this add-on at this version* whereas the compare title is conveying *compare this add-on between version1 and version2*.

Reply:
Well, I guess it doesn't matter. I can make a colon since that makes more sense to everyone.